src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -327,4 +327,22 @@ export default class ReimbursementRequestService {
 
     return reimbursementRequests.map(reimbursementRequestTransformer);
   }
+
+  /**
+   * Gets a single reimbursement request for the given id
+   * @param reimbursementRequestId the id of thereimbursement request to get
+   * @returns the reimbursement request with the given id
+   */
+  static async getSingleReimbursementRequest(reimbursementRequestId: string): Promise<ReimbursementRequest> {
```

Comment:
this should have permissions (and now that I think about it, getAllReimbursementRequests should too). This should only allow people on the finance team or the original submitter of the request to get it. Can you also update getAll to only allow ppl on the finance team to hit it. Since we're dealing with finance data for this stuff, I wanna be more particular about who can get what